packages/mask/src/plugins/Referral/SNSAdaptor/shared-ui/icons/AttraceLogoText.tsx

```diff
@@ -0,0 +1,23 @@
WcbAtBs7iE2ReL2fQ6gEQ03Tp9xTasP+k+RNugds21gzitvQNN3nkmbqdWNVQRYa3eId086SI109CUNwYoJNdsFIBhQa/W6esGcTDd/rpq1a3EEGVWRc7Jdd40qL9LXW15ru+5t0hQ7T2yzbbAsCiEwLs93LDAXApDl+8WO+VyxY3652DHvNx9WmXaJLwrJvrnvSvnNfVeY6XY3O5O6m53J383OlJezM0LDqG5ppn+d+b5Y/EBhpRfym+pRQg5pJieWo/39ecoZx8Iru3tX7o+df9eDyFDoBs8zVdNB/zm+NLGWb+v9ecqZ2/eZaWa9/9Bm32nXOvDQ1Aipm8Nm28CcV/6CE5MOWBqrQ3/k310yTEvn/hfp60dCABlQb8/SgWfOmZ3Y/Sb5cBNzvFtsDk3KmGsgwor7fIkxZuw0zs6aQ8t72XeGSc8GQxo45xiq15Y5tBZAcH++Y2ELVeajlPdTdLWUvyaEqu3Ggc2lvVfZdonPoZEC/86b+65sgDZ/qHoTaWS8x4qonW5ppn+d+X6QNFOBtrzSNX4ZATpiW878zaF5zROTDe5nDgIjTUF/f5s5NC96ZPbkpjrciMoYqnN7ds9UeY9+QKRuW8u3dg+x8xvdvg+sNLPen8uLfWhqpPBB/6kMBY7lvqAr3xLOQ3MtRwGrlbUohOmZYV3jzOwx8HtosYXZsTsdm4AVd1Wo5wv6NFHTwesS/XVr79F0f45eoO4eBFxptWeE0sbjOuQIA1pj3dOHN9BEK18aMEpr7+neF5tfY6alNc30rzuV46CUXsjn1XBN29ND86pHZk+uUg+N65mq6aD/XE15aJ2882b08610vo15X3b7LVjOBNqrgNSJJsaY0wAWmA3ufrORbuhzNtYdaKUzrz8Z62PLMaBl+waQG2gNl9uue5gDQtVhO3lAc2y8ix3zfvclrTraJWljta6Si6x+ctlXCLECmnpqM1mOnQBNRI/MniQd5M5tALTOT0yfEXIGPj41QoexrVMYYM6r2gRaCU2M2aYJyzPTeWVVeQBafIcYqY7faV9WTgP2DzQvCxrKXguIH6D5EWYczREC0IQkA2hhiMyfkeESrDQ8CdBEtA2AdlkUZJo+PjVCjofJ1CHQCMgc21kWzAx7HDwCbeOl158MZEVYAFp2bgC5Cn368p7rQkNifUvZVTD3zIMHmt8d/uXqb4UDtH+fnRH2Jv0DLRylF/Kr1EPbVkCToToGWucnp4d9RSipQ6C5ggzMMBvc/WaGNbTjAWix98p0ovHffAPNyyIQulIpNKD5ClDbcaOHLIVdCRtovLLcPwGaTKBtAEb3mHml+34CNPM1N9BIUGUCMiltpw5otT6HxgUyYHhmtl8SBFrNwOz4nfb1aob7ApqXocZxfaEHDwVEEGgyoq0X9fcNAWhC3qQMoIUtCrTmGC8KWcNa9BYZHppEUaDtrPFl+2RVb+ZTkkCmSQe0FbNtYM6reAKtkxdkoIcZXfjhuDKPE2j3Rl7fLRRSJwaqRPHwAbSm8T03hFY29i9lSTpdDhFovgOJdtzoKZp7ewEDzUsAVuP9awNom9RDK8UMaGTFb+Zzc8c3qzb4A5ps0VWOKQCj51gjQLtMbP/U9HAg+3HJopCHpkYyIexDY3zGM9AI3Hc6LfZgSe+ZcYUd4QBaLQ0xKtKfOQX+gOZlmX6B9nypggaaf7FWTAUINO65SMf7CwAtKhGgpRfyJGo9GTpjbKqvXoQANPL7nb85dzxHQGa1IXZA23xk9iTZh/ZlbDgaxWgTGP6OFGjExsc/PVPIf3q6EPiqb7IPTfVgq9MEVftiBTQC3RSaGBOGux5m3JOODsZunX1jd+jBb/3quTvthjOnwDvQsuN7bgh5Ev1L2U2gY9shAE3K7vy9N3qYE8wBAU1oYU2tAw1UqBUx9TQgGqCRYanUb84dZ3ZM4wo0UKG2/MjsC2TIdljy8TGG7/sAGhkO7fzMTKH5MzOFUBfJkUgh1Evr1AMf4gE0UuYeRhNjBd4o+WYpMBvc/Waz/jRpHtkYW3NemSZsCm8F3oEmPHemA1opaKDJEgVayjweHwDQBDoHHPevHaCtf2khTzyNx8296QCBRhq4L39+7ljm84zo+2wb4gc0UKD2QoF69Z3kuWIANNJZ7fzszFDzZ2cKkbaTD02NFB+aGmnGNG201yMCGhnGfhxNjGXcouK7SQtn5WkpKLaGNfEUQT4Oev5Oe/n807dIxjbpsw3bhuWxDY+TB9OheTyiQMtNtZUIVAtqyCfd/TF/KJnqtSH01RqSPAS8V13dmLm591pOOaqBRu52tEsszNI9JHzMDTNPjPe3zU/QvRIPfWkhT3rvmbe6LpH5NHIUStZgr5zQVyvk+JjH5o65lg9mmgmGvgpLj6inVZNnKv7Pg2dTNP0sofoq9hrqGEc5cg99taFu/YHiL8wMxS5GLYEaSZu/U8Ng5dR9j6ihklfBhb4iYFeO3fELML2UqPmDu9/c9BOmnxq7NvrGE6EevS1b55++RcaVhxCjSbOPBA6mvominUdv7/VVeKfaSvScKeWIieoxHUwbzP6cog0a4boMgMs9t7oCX5Rzc+81kv+VBsMx0jg7svsaDWhcRoCXd1/vjXUEmSj0VtelRlQ9WiNTOZhRPNq+dlTM8m/NP1trC7Z86c8PnqVHt5D0wyn9diRrHWOWU9214f2NSmR5BOVfnGEfIRNnUbCRdMkgbVrCX7R9MLZFUP7oxGgg5Q09/8QbKS+ehOVGAOOjbzxRc/Nlep1/+lazNlTqE2jFo7f3SvWCptuU06nVoxbsgaaMwR+81Rn5hvVbe68px0HQxqIR2EDbpJV/86nrB2uu4sdBf9B1qZkGlU4BImdsYUOHkqZ5JZ0B8Opvzz9bKwENQtG7B0e0o0tomVUSLkXPLLMDmpaG5AR90jiv/pLHwz3jrP/b/4KaLkjpOBEp5+HZAE0rZyTNSH3efHByNJyyBgD/H05k3bZoPCICAAAAAElFTkSuQmCC"
```

Comment:
It's a bitmap image.

Reply:
I will discuss with a designer to provide vector images.